0001-create-schema.sql

```diff
@@ -0,0 +1,12 @@
+-- auth schema
+CREATE SCHEMA IF NOT EXISTS auth;
+-- https://github.com/hasura/graphql-engine/issues/3657
+CREATE EXTENSION IF NOT EXISTS pgcrypto WITH SCHEMA public;
+CREATE EXTENSION IF NOT EXISTS citext WITH SCHEMA public;
+CREATE OR REPLACE FUNCTION public.set_current_timestamp_updated_at() RETURNS trigger LANGUAGE plpgsql AS $$
+declare _new record;
+begin _new := new;
+_new."updated_at" = now();
+return _new;
+end;
+$$;
```

Comment:
I don't understand why this eas added as a new file in this PR 🤔

Reply:
It's in the repo's root